autoscale/src/main/java/com/sequenceiq/periscope/controller/AlertController.java

```diff
@@ -265,30 +280,27 @@ private PrometheusAlertResponse createPrometheusAlertResponse(PrometheusAlert al
         return prometheusAlertResponseConverter.convert(alarm);
     }
 
-    private void validateAlertForUpdate(Long clusterId, Optional<Long> alertIdRequest, AlertType alertType) {
-        alertIdRequest.ifPresent(alertId -> {
-            BaseAlert alert;
-            String clusterCrn = clusterService.findStackCrnById(clusterId);
-            switch (alertType) {
-                case LOAD:
-                    alert = alertService.findLoadAlertByCluster(clusterId, alertId);
-                    break;
-                case TIME:
-                    alert = alertService.findTimeAlertByCluster(clusterId, alertId);
-                    break;
-                case METRIC:
-                    alert = alertService.findMetricAlertByCluster(clusterId, alertId);
-                    break;
-                case PROMETHEUS:
-                    alert = alertService.findPrometheusAlertByCluster(clusterId, alertId);
-                    break;
-                default:
-                    alert = null;
-
-            }
-            if (alert == null) {
-                throw new NotFoundException(String.format("Could not find %s alert with id: '%s', for cluster: '%s'", alertType, alertId, clusterCrn));
-            }
-        });
+    private void validateAlertForUpdate(Long clusterId, Long alertId, AlertType alertType) {
+        BaseAlert alert;
+        String clusterCrn = clusterService.findStackCrnById(clusterId);
```

Comment:
Nit: Move into the (alert == null) block.

Reply:
ack